src/common/snippets/include/snippets/lowered/loop_info.hpp

```diff
@@ -309,6 +319,14 @@ class UnifiedLoopInfo : public LoopInfo {
      */
     void replace_with_new_ports(const ExpressionPort& actual_port, const std::vector<ExpressionPort>& target_ports) override;
 
+    /**
+     * @brief Remove remove_ports and add add_ports to the current LoopPort.
+     *        This function remove and add directly without respect ports order, caller is responsible for the order by sort.
+     * @param remove_ports need to be removed
+     * @param add_ports need to be added
+     * @param is_input true if update input port
+     */
+    void update_loop_ports(const std::vector<ExpressionPort>& remove_ports, const std::vector<ExpressionPort>& add_ports, bool is_input);
```

Comment:
Just note:
As we discussed offline, we should implement one common method `replace_with_new_loop_ports(vector, vector` instead of existing 3 methods. For that, we need to implement CVS-142990.